DESCRIPTION

```diff
@@ -1,5 +1,5 @@
 Package: BoutrosLab.plotting.general
-Version: 7.0.4
+Version: 7.0.5
 Type: Package
 Title: Functions to Create Publication-Quality Plots
 Date: 2022-03-15
```

Comment:
I updated the date here. (CRAN checks will fail if the date field is too far in the past.)